<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceClass</class>
 <widget class="QDialog" name="AttendanceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>505</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Attendance</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QScrollArea" name="children_scroll">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="children_scroll_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>694</width>
        <height>298</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>165</height>
      </size>
     </property>
     <property name="selectedDate">
      <date>
       <year>2027</year>
       <month>7</month>
       <day>26</day>
      </date>
     </property>
     <property name="gridVisible">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::SelectionMode::SingleSelection</enum>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::HorizontalHeaderFormat::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::VerticalHeaderFormat::ISOWeekNumbers</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="header">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>165</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="2" colspan="2">
       <widget class="QScrollArea" name="employees_scroll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="employees_scroll_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>178</width>
           <height>155</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="timeofday">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time Of Day</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="weekday">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Week Day</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="accept_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="cancel_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>AttendanceClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>AttendanceClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
